.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+# Auto formats python code
```

Comment:
Move this comment to be above the "black" repo item, not at the top of the list.

And capitalize Python.